test/assets/config-override.jsonld

```diff
@@ -0,0 +1,31 @@
+{
+  "@context": {
+    "@vocab": "https://linkedsoftwaredependencies.org/vocabularies/object-oriented#",
+    "ex": "http://example.org/",
+    "hello": "http://example.org/hello/"
+  },
+  "@graph": [
+    {
+      "@id": "ex:myHelloWorldWithOverride",
+      "@type": "ex:HelloWorldModule#SayHelloComponent",
+      "hello:hello": "WORLD",
+      "hello:say": "HI"
+    },
+    {
+      "@id": "ex:myObjectOverride",
+      "@type": "Override",
+      "overrideInstance": { "@id": "ex:myHelloWorldWithOverride" },
+      "overrideParameters": {
+        "hello:hello": "BETTER WORLD"
+      }
+    },
+    {
```

Comment:
Could you place this one before `ex:myObjectOverride`, just so we test out of order overrides as well?
(or perhaps create a separate out of order variant if this test)